src/main/java/no/found/elasticsearch/example/TransportExample.java

```diff
@@ -38,6 +38,8 @@
 public class TransportExample {
 
     public ESLogger logger = ESLoggerFactory.getLogger(getClass().getCanonicalName());
+    // Note: if enablng IPv6 then should ensure that your host and network can route it to the Cloud endpoint
```

Comment:
Note: If enabling IPv6, then you should ensure that your host and network can route it to the Cloud endpoint.